apps/sms/test/marionette/generators/thread.js

```diff
@@ -0,0 +1,54 @@
+function ThreadGenerator() {};
+
+ThreadGenerator.uniqueThreadId = 0;
+ThreadGenerator.uniqueMessageId = 0;
+
+ThreadGenerator.injectDependencies = function(client) {
+  client.contentScript.inject(
+    __dirname + '/../mocks/mock_test_storages.js'
+  );
+  client.contentScript.inject(
+    __dirname + '/../mocks/mock_navigator_moz_mobile_message.js'
+  );
+};
+
+ThreadGenerator.generate = function(numberOfMessages, messageType, participants) {
```

Comment:
* Line is too long;

* I think consumer code will be easier to read if you pass parameters as single object with properties.